apps/balancer-tools/src/app/apr/round/(components)/PoolListTable.tsx

```diff
@@ -1,45 +1,75 @@
-import { ChevronDownIcon } from "@radix-ui/react-icons";
```

Comment:
Good call on this, that's the only way to make it feel really dynamic and if the client's IP get rate limited he should eventually make this request